documentation/2.5.1/classsf_1_1Image.php

```diff
@@ -423,7 +423,7 @@
 </div><div class="memdoc">
 
 <p>Get a read-only pointer to the array of pixels. </p>
-<p>The returned value points to an array of RGBA pixels made of 8 bits integers components. The size of the array is width * height * 4 (<a class="el" href="classsf_1_1Image.php#a85409951b05369813069ed64393391ce" title="Return the size (width and height) of the image. ">getSize()</a>.x * <a class="el" href="classsf_1_1Image.php#a85409951b05369813069ed64393391ce" title="Return the size (width and height) of the image. ">getSize()</a>.y * 4). Warning: the returned pointer may become invalid if you modify the image, so you should never store it for too long. If the image is empty, a null pointer is returned.</p>
+<p>The returned value points to an array of RGBA pixels made of 8 bits integers components. The size of the array is width * height * 4 (<a class="el" href="classsf_1_1Image.php#a85409951b05369813069ed64393391ce" title="Return the size (width and height) of the image. ">getSize()</a>.x * <a class="el" href="classsf_1_1Image.php#a85409951b05369813069ed64393391ce" title="Return the size (width and height) of the image. ">getSize()</a>.y * 4). Warning: the returned pointer may become invalid if you modify the sf::Image (though only modifying its data does not invalid the pointer) so you should never store it for too long. If the image is empty, a null pointer is returned.</p>
```

Comment:
We aren't manually editing the documentation, as it's generated automatically from source with doxygen. Unfortunately that also means that unclear things in the documentation will stay like they are until the next version, but we really don't want to have differences between the documentation in the matching code and the documentation on the website.

Reply:
Reverted the documentation change.